src/niclips/figures/multi_view.py

```diff
@@ -52,16 +59,21 @@ def multi_view_frame(
             fontsize=fontsize,
         )
 
-        if overlay is not None:
-            panel_overlay = noimg.render_slice(
-                overlay,
-                axis=axis,
-                coord=coord,
-                height=panel_height,
-                cmap=overlay_cmap,
-                fontsize=fontsize,
-            )
-            panel = noimg.overlay(panel, panel_overlay, alpha=alpha)
+        if len(overlay) > 0:
+            if len(overlay_cmap) < len(overlay):
+                logging.warning(
+                    "More overlays than overlay color maps- will use 'turbo'",
+                )
+            for ov, ov_cmap in zip_longest(overlay, overlay_cmap):
+                panel_overlay = noimg.render_slice(
```

Comment:
The image will be annotated with XYZ coordinates by default. Not sure we need this for the overlays.

Reply:
I meant how `render_slice` will add a text annotation to the image (`annotate=True` by default). In the case of overlays, this will just keep drawing e.g. `X=42, Y=10, Z=8` in the same spot.

Reply:
I'll take a look - if it's already annotated once, we can have it turn off for the others.